include/deal.II/grid/manifold_lib.h

```diff
@@ -503,6 +503,60 @@ class TorusManifold : public ChartManifold<dim,3,3>
   double r, R;
 };
 
+
+
+/**
+ * A mapping class that smoothly transitions from different manifold
+ * descriptions on different faces by a so-called transfinite
+ * interpolation. As opposed to setting the manifold description only on the
```

Comment:
If I understand this correctly, what you're doing is taking a curved description for one or more faces of a cell, and then construct some kind of smooth continuation of this description into the interior. Is this correct? If so, would you add a sentence or two here that explains how this is actually done?